src/main/java/seedu/address/model/order/Price.java

```diff
@@ -34,7 +34,7 @@ public Price(String value) {
      * @return true if the price is valid
      */
     public static boolean isValidPrice(String test) {
-        return Double.parseDouble(test) > 0;
+        return Double.parseDouble(test) >= 0;
```

Comment:
Did you check small negative values eg -0.005 which may have similar issue to 0.005

Reply:
Done, checked orders with postiive, 0, negaitve and -0.005, works great now.